### ISSUE TITLE
NameError: name 'get_modification_time' is not defined

### ISSUE THREAD

**Issue by rohanbanerjee** [spinalcordtoolbox/manual-correction#93]
I am using the latest version of the `manual-correction` software, SHA - `7242702d1c8d40e5ea820d2b4c4cc0c852e49c43`.
I am getting the error `NameError: name 'get_modification_time' is not defined` while running it. I do understand the functionality of it but I'm curious why it was included there without defining the function or am I missing out on something here. 

If it was not defined then I can make a function `get_modification_time` in `utils.py` and call it here.

**Comment by NathanMolinier** [comment]
It looks like this error was introduced by the last commit 7242702d1c8d40e5ea820d2b4c4cc0c852e49c43 and you seem to be the co-author here ahah

Related to this PR: https://github.com/spinalcordtoolbox/manual-correction/pull/88

**Comment by valosekj** [comment]
Thanks for reporting this, @rohanbanerjee. I have a feeling that https://github.com/spinalcordtoolbox/manual-correction/pull/88 was not properly tested before merging. Especially after this merge commit (https://github.com/spinalcordtoolbox/manual-correction/pull/88/commits/39acd0fafe0ab50edacfb44691f07d6a3bdde79b).

I can still see an option to revert the PR:

<img width="974" alt="image" src="https://github.com/spinalcordtoolbox/manual-correction/assets/39456460/9e2de17b-b6c3-4056-b4e1-60683daa56ec">

So maybe we can revert the merge, which will make `main` work again. And after proper testing, the PR could be remerged. Any ideas?

**Comment by rohanbanerjee** [comment]
Yup that sounds good. I have opened a PR to revert the changes. Will make fresh changes and open another PR for the same.

**Comment by valosekj** [comment]
Reverted now by https://github.com/spinalcordtoolbox/manual-correction/pull/94
